curriculum/challenges/english/25-front-end-development/lecture-working-with-functions/672d269da46786225e3fe3fd.md

```diff
@@ -10,6 +10,95 @@ dashedName: what-is-the-purpose-of-functions-and-how-do-they-work
 
 Watch the video lecture and answer the questions below.
 
+# --transcript--
+
+What is the purpose of functions, and how do they work?
+
+Functions are reusable pieces of code that perform a specific task or calculate a value. Think of functions as a machine that takes some input, does some operations on it, and then produces an output. Here is an example of declaring a function:
+
+```js
+function greet() {
+  console.log("Hello, Jessica!");
+}
+```
+
+In this example, we have declared a function called `greet`.  Inside that function, we have a `console.log` that logs the message `Hello, Jessica!`. If we tried to run this code, we would not see the message appear in the console. This because we need to call the function.
```

Comment:
```suggestion
In this example, we have declared a function called `greet`.  Inside that function, we have a `console.log` that logs the message `Hello, Jessica!`. If we try to run this code, we would not see the message appear in the console. This because we need to call the function.
```